Benchmarks/Benchmarks/Basic/BenchmarkRunner+Basic.swift

```diff
@@ -125,7 +125,7 @@ let benchmarks: @Sendable () -> Void = {
         }
     }
 
-    let parameterization = (0...5).map { 1 << $0 } // 1, 2, 4, ...
+    let parameterization = (0...5).map { 1 << $0 }  // 1, 2, 4, ...
 
```

Comment:
Please merge in `.swift-format` from main and apply `swift-format`, then we can remove a lot of these diffs with extra spaces.